telethon/client/updates.py

```diff
@@ -459,10 +459,10 @@ def _process_update(self: 'TelegramClient', update, others, entities=None):
 
             if not pts_date:
                 # First-time, can't get difference. Get pts instead.
-                result = await self(functions.messages.GetPeerDialogsRequest([
-                    utils.get_input_dialog(where)
-                ]))
-                self._state_cache[channel_id] = result.dialogs[0].pts
+                result = await self(functions.channels.getFullChannel(
```

Comment:
```suggestion
                result = await self(functions.channels.GetFullChannelRequest(
```

…as showcased in https://tl.telethon.dev/methods/channels/get_full_channel.html.

Reply:
That's what I get for making fast PRs without testing :p